build-system/tasks/serve.js

```diff
@@ -23,6 +23,18 @@ var nodemon = require('nodemon');
  * Starts a simple http server at the repository root
  */
 function serve() {
+  // Get the serve mode
+  if (argv.compiled) {
+    process.env.SERVE_MODE = 'min';
```

Comment:
I can add the check for `app.use('/serve_mode=?')` to change it at runtime. If this is what you mean?